calblueprint/shanti-project
# monique/confirmation-pages-backend

## What's new in this PR
### Description
- Added backend to order confirmation pages for Delivery and Pickup
- Centered the order confirmation box

### Screenshots

## How to review
Testing:
- go to [localhost](http://localhost:3000/)
- login (might have to temporarily change delivery_allowed to TRUE to see the delivery page)
- go to cart > checkout and you will be redirected to the confirmation page

## Next steps
n/a

🛍 CC: @BuyankhuuTsCAl
